docs(uninstall): add cluster-wide-resources uninstall docs (#271)
openservicemesh/osm-docs: content/docs/getting_started/cleanup.md

kubectl delete ns bookbuyer bookthief bookstore bookwarehouse
```

# Uninstalling the OSM Control Plane

To uninstall OSM, run the following command.
```bash
osm uninstall mesh
```

> > Note: `osm uninstall mesh` does not delete the namespace where osm is installed. This prevents unintended consequences, such as if osm was installed in `kube-system`, as deleting that namespace may delete important cluster resources and may have unintended consequences.

Reviewer: I'd also change "consult with the..." to "consult the..."

> Note: `osm uninstall mesh` does not fully uninstall all osm resources to prevent unintended consequences. Consult with the [uninstallation guide](/docs/guides/uninstall/) for more context and information.

# Fully Cleaning Up OSM Resources

After uninstalling OSM, to fully clean up all OSM resources (including the meshconfig, secrets, OSM CRDs, SMI CRDs, and webhook configurations), run the following command.

> Consult with the [uninstallation guide](/docs/guides/uninstall/) for more context and information.

Reviewer: ```suggestion
Refer to the [uninstallation guide](/docs/guides/uninstall/) for more context and information.
```
